week3/assign1/src/components/Button.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+
+const Button =(props)=>{
```

Comment:
여러 개의 props를 내려 받을 때는 `구조분해할당`을 적극적으로 사용하는 것을 추천합니다!
```suggestion
const Button =({onClick, state, content})=>{
```
-> 이런식으로 가능해용 ~

Reply:
마자 구조분해할당 손에 한번 익으면 이전으로 못돌아가 ~.~